### ISSUE TITLE
Using Envato Toolkit on ThemeForest Alaska Theme results in Error

### ISSUE THREAD

**Issue by ragtech** [envato/envato-wordpress-toolkit#70]
Warning: function_exists() expects parameter 1 to be string, array given in /home/ragtech/public_html/wordpress/wp-admin/includes/class-pclzip.php on line 1766

**Comment by adele72** [comment]
Were you able to fix this?  I ran into the same problem.

**Comment by ragtech** [comment]
No, never did... I just did it the old fashion way; deleted the theme folder and re-up'd the new version of the theme.
